themes/spichka/resources/styles/front-page.scss

```diff
@@ -0,0 +1,74 @@
+.section-banner-video {
+  height: 100vh;
+  align-items: center;
+  display: flex;
+
+  .banner-video {
+    position: absolute;
+    inset: 0;
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    filter: brightness(60%);
+  }
+
+  .header {
+    color: white;
+    position: relative;
+    z-index: 2;
+
+    p {
+      font-weight: 800;
+    }
+  }
+}
+
+// Donate
+.donate-section {
+  background-color: #333;
+  padding: 3rem 0;
+  color: #fff;
+
+  img {
+    max-width: 100%;
+  }
+
+  .btn {
+    padding: 0.5rem 1rem;
+  }
+}
+
+// Connect
+.connect {
+  .connect-card {
+    background-color: #3b3d42;
+    color: white;
+    padding: 4.5rem 1rem;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+
+    .btn {
+      margin: 1rem 2rem 0;
+      margin-top: auto;
+      text-decoration: none;
+    }
+  }
+
+  // Carousel
+
+  .swiper-connect.stretch-cards {
+    .swiper-wrapper {
+      // align-items: stretch;
+    }
+
+    .swiper-slide {
+      > * {
+        height: 100%;
+      }
+
+      height: unset;
+    }
+  }
```

Comment:
Im not sure that stratching works properly now with carousel

Reply:
Fix it ;]